cpp/ql/lib/experimental/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -859,6 +890,8 @@ abstract class PostUpdateNode extends Node {
    * Gets the node before the state update.
    */
   abstract Node getPreUpdateNode();
+
+  final override Type getType() { result = this.getPreUpdateNode().getType() }
```

Comment:
Is there a reason this returns a `Type` and not a `DataFlowType`?

Reply:
Oh. Good point. It should really return a `DataFlowType` (it doesn't really matter since the types are equal, but we should really be consistent).

Reply:
Fixed in 5cfc3fe8df5b37daae06755e8f1acae664e15f41.